portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,124 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Fiana Vicker's Professional Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body = style = "border-top: lightslategray;
+                   background-color: grey; background-image: url('images/wooddark.jpeg');
+                   
+                   " >
+
+    <div id="banner">
+        <h2 style="top: 0px; background-color: palevioletred; color: lightgray;">
+                Personal Projects        |        Gallery       |          Professional Projects      
+                   |         Resume        |  Instagram  | 
+                   
+                   <a href="https://github.com/fianaVickers">GitHub </a> |  
+                   <a href="https://www.linkedin.com/in/fiana-vickers-ba51a3170/">LinkedIn  </a> </h2>
+    </div>
+
+
+    <div id="content">
+        
+      <h1 = style="margin-top: 0.25in; font-size: 65pt; color: lightgrey; font-style: italic;"> Fiana Vickers </h1>
```

Comment:
The `=` between `h1` and `style` shouldn't be there. It's just `<h1 style="... your stuff here ...">` 

This can also be moved to your .css file.